### DISCUSSION TITLE
August 4th Generation via August Connect Bridge Work?

### DISCUSSION THREAD

**Discussion by aperture01** [homebridge-plugins/homebridge-august#136] [Q&A]
Hi All,

I just recently discovered this plugin despite having this lock for some time. I recently purchased the connect bridge even though this lock has wifi built in because i noticed the battery life was getting pretty bad. This didn't improve my homekit performance as I eventually learned the lock uses bluetooth from the home hub to communicate, and the bridge only works from the august app. For this reason the lock fails regularly from apple home, as the nearest home hub is still a good distance away. 

I've installed this plugin in my homebridge environment and I get a message "Total August Locks Found: 1" after the initial sign in and config. However this lock does not appear in Apple Home Devices, and I don't know where to find the "Lock ID". I've added the bridge to home already so I figure i'd see it. 

I was thinking of removing it from the august connect bridge and connecting it natively via its own Wi-Fi again. Before I do that does anyone know if my config should work as is, or do I need to remove it from the bridge to get it to work with this? Any guidance would be appreciated.

Thanks for the wonderful plugin! Hope to give it a try soon.

Thanks!

**Comment by ZiIlaGit104** [comment]
I have the same issue.... My locks are supported and the log shows:
[6/26/2025, 10:14:23 PM] [August] Launched child bridge with PID 9151
[6/26/2025, 10:14:25 PM] Registering platform '@homebridge-plugins/homebridge-august.August'
[6/26/2025, 10:14:25 PM] [August] Loaded @homebridge-plugins/homebridge-august v3.0.2 child bridge successfully
[6/26/2025, 10:14:25 PM] Loaded 0 cached accessories from cachedAccessories.0E773D24ECE5.
[6/26/2025, 10:14:25 PM] Homebridge v1.10.0 (HAP v0.13.1) (August) is running on port 58145.
[6/26/2025, 10:14:26 PM] [August] Total August Locks Found: 2


But.... no locks are ever imported and I cannot see a LockID to add to he config.

**Reply by LewisKarkov** [reply]
In Settings, change Logging Setting to Debug Logging. You'll get the LockID and the reason why they don't show up as accessories.

Possible reason (maybe also applicable for OP): you already have it in Homekit and the plugin doesn't make it available as accessories. Either remove them from Homekit or use "Override HomeKit Enabled" in "August Device Settings" (LockID needed).

**Reply by ZiIlaGit104** [reply]
That did the trick! Needed the debug logging to see the LockID... pretty obvious now.  Thanks!!

**Comment by vit1999-gif** [comment]
Hello! Please help. I have added 2 locks, but i see only one Yale lock, august lock doesn't seen in homebridge
There is all logs
Total August Locks Found: 3
[10/14/2025, 4:03:15 PM] [August] Adding new accessory: Веранда, Lock ID: #############
[10/14/2025, 4:03:15 PM] [August] Lock: Веранда (refreshStatus) lockDetails: No details
<img width="957" height="872" alt="Снимок экрана 2025-10-14 в 16 05 24" src="https://github.com/user-attachments/assets/c5e598a9-3b9d-4e91-bbbe-f24e8092d4a2" />
<img width="986" height="903" alt="Снимок экрана 2025-10-14 в 16 05 15" src="https://github.com/user-attachments/assets/2aea3610-f2dd-41b0-83ba-6891d5ee1eb3" />
